src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar">
```

Comment:
please implement this part of the task
<img width="804" alt="Screenshot 2023-03-24 at 21 19 45" src="https://user-images.githubusercontent.com/32853034/227630643-b437a94d-4142-4382-9edc-e68a8e0ac9e7.png">

For example:
```suggestion
    <ul class="calendar calendar--start-day--mon calendar--month-length--30">
```